From 59141878c8a9df3f28ffbea1df1a8108018a89b3 Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@cloudera.com>
Date: Mon, 6 Jan 2014 15:40:21 -0800
Subject: [PATCH 121/596] MR1: MAPREDUCE-3310. Addendum fixing javadocs
 warnings (tucu)

(cherry picked from commit 820a059a60f6eb42887db77df0286a8d2695b6b4)
(cherry picked from commit 2c035f344f5d36e1c17b1445021844f7623c78e1)
(cherry picked from commit fd3dd50b776e51c4987391bdcd5100bcdd55a1d8)
---
 .../mapred/org/apache/hadoop/mapred/JobConf.java   |    7 +++----
 .../mapred/org/apache/hadoop/mapreduce/Job.java    |    5 ++---
 .../org/apache/hadoop/mapreduce/JobContext.java    |    5 ++---
 3 files changed, 7 insertions(+), 10 deletions(-)

diff --git a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/JobConf.java b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/JobConf.java
index 7118d6e..7766311 100644
--- a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/JobConf.java
+++ b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/JobConf.java
@@ -888,7 +888,7 @@ public String getKeyFieldPartitionerOption() {
    * grouping keys of inputs to the combiner.
    *
    * @return comparator set by the user for grouping values.
-   * @see #setCombinerKeyGroupingComparator(Class) for details.
+   * @see #setCombinerKeyGroupingComparator(Class) 
    */
   public RawComparator getCombinerKeyGroupingComparator() {
     Class<? extends RawComparator> theClass = getClass(
@@ -906,7 +906,7 @@ public RawComparator getCombinerKeyGroupingComparator() {
    * grouping keys of inputs to the reduce.
    * 
    * @return comparator set by the user for grouping values.
-   * @see #setOutputValueGroupingComparator(Class) for details.  
+   * @see #setOutputValueGroupingComparator(Class)
    */
   public RawComparator getOutputValueGroupingComparator() {
     Class<? extends RawComparator> theClass = getClass("mapred.output.value.groupfn.class", null,
@@ -974,8 +974,7 @@ public void setCombinerKeyGroupingComparator(
    * @param theClass the comparator class to be used for grouping keys. 
    *                 It should implement <code>RawComparator</code>.
    * @see #setOutputKeyComparatorClass(Class)
-   * @see {@link #setCombinerKeyGroupingComparator(Class)} for setting a
-   * comparator for the combiner.
+   * @see #setCombinerKeyGroupingComparator(Class)
    */
   public void setOutputValueGroupingComparator(
 		  Class<? extends RawComparator> theClass) {
diff --git a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/Job.java b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/Job.java
index 020ce2c..41ee8a5 100644
--- a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/Job.java
+++ b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/Job.java
@@ -320,7 +320,7 @@ public void setOutputValueClass(Class<?> theClass
    * are passed to the {@link Reducer}.
    * @param cls the raw comparator
    * @throws IllegalStateException if the job is submitted
-   * @see {@link #setCombinerKeyGroupingComparatorClass(Class)}
+   * @see #setCombinerKeyGroupingComparatorClass(Class)
    */
   public void setSortComparatorClass(Class<? extends RawComparator> cls
                                      ) throws IllegalStateException {
@@ -350,8 +350,7 @@ public void setCombinerKeyGroupingComparatorClass(
    *                       org.apache.hadoop.mapreduce.Reducer.Context)}
    * @param cls the raw comparator to use
    * @throws IllegalStateException if the job is submitted
-   * @see {@link #setCombinerKeyGroupingComparatorClass(Class)} for setting a
-   * comparator for the combiner.
+   * @see #setCombinerKeyGroupingComparatorClass(Class)
    */
   public void setGroupingComparatorClass(Class<? extends RawComparator> cls
                                          ) throws IllegalStateException {
diff --git a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/JobContext.java b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/JobContext.java
index 88f5955..341efad 100644
--- a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/JobContext.java
+++ b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapreduce/JobContext.java
@@ -222,7 +222,7 @@
    * grouping keys of inputs to the combiner.
    *
    * @return comparator set by the user for grouping values.
-   * @see Job#setCombinerKeyGroupingComparatorClass(Class) for details.
+   * @see Job#setCombinerKeyGroupingComparatorClass(Class)
    */
   public RawComparator<?> getCombinerKeyGroupingComparator();
 
@@ -232,8 +232,7 @@
    * 
    * @return comparator set by the user for grouping values.
    * @see Job#setGroupingComparatorClass(Class) for details.  
-   * @see {@link #getCombinerKeyGroupingComparator()} for setting a
-   * comparator for the combiner.
+   * @see #getCombinerKeyGroupingComparator()
    */
   public RawComparator<?> getGroupingComparator();
   
-- 
1.7.9.5

